modules/@ergonode/authentication/src/components/Forms/NewPasswordForm.vue

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <LoginForm
+        :submit-title="$t('authentication.forms.newPassword.button')"
+        :is-submitting="isSubmitting"
+        :errors="errors"
+        @submit="onSubmit">
+        <template #header>
+            <FormHeader :text="$t('authentication.forms.newPassword.title')" />
+        </template>
+        <template #body>
+            <div class="new-password-form__info">
+                <FormParagraph :text="$t('authentication.forms.newPassword.info')" />
+                <List>
+                    <ListElement
+                        v-for="(value, key) in validationHelper"
+                        :item="value"
+                        :key="key" />
+                </List>
+            </div>
+            <TextField
+                v-model="checkPassword"
+                required
+                :input="passwordInputType"
+                :error-messages="errors[passwordFieldKey]"
+                :label="$t('authentication.forms.newPassword.field')"
+                @keydown.enter.prevent="onSubmit" />
+            <Toggler
+                v-model="isPasswordVisible"
+                :label="$t('authentication.forms.newPassword.toggle')" />
+        </template>
+    </LoginForm>
+</template>
+
+<script>
+import LoginForm from '@Authentication/components/Forms/LoginForm';
+import ListElement from '@Authentication/components/Lists/ListElement';
+import {
+    LOGIN_STATE,
+} from '@Authentication/defaults/login-state';
+import {
+    ROUTE_NAME,
+} from '@Core/config/routes';
+import {
+    ALERT_TYPE,
+} from '@Core/defaults/alerts';
+import scopeErrorsMixin from '@Core/mixins/feedback/scopeErrorsMixin';
+import FormHeader from '@UI/components/Form/FormHeader';
+import FormParagraph from '@UI/components/Form/FormParagraph';
+import List from '@UI/components/List/List';
+import TextField from '@UI/components/TextField/TextField';
+import Toggler from '@UI/components/Toggler/Toggler';
+import {
+    mapActions,
+} from 'vuex';
+
+export default {
+    name: 'PasswordRecoveryForm',
+    components: {
+        FormHeader,
+        FormParagraph,
+        LoginForm,
+        ListElement,
+        List,
+        Toggler,
+        TextField,
+    },
+    mixins: [
+        scopeErrorsMixin,
+    ],
+    data() {
+        return {
+            password: '',
+            isPasswordVisible: false,
+            isSubmitting: false,
+            maxLength: 6,
+
+        };
+    },
+    computed: {
+        checkPassword: {
+            get() {
+                return this.password;
+            },
+            set(value) {
+                this.validationHelper.hasMinimumLength = {
```

Comment:
computed property should not be mutated

Reply:
As I said earlier, there was a problem rendering i18n placeholders if they were in `data` section. that's why `validationHelper` i computed.